.github/workflows/release.yml

```diff
@@ -1,32 +1,32 @@
-# name: Release CI
-# on:
-#   push:
-#     branches:
-#       - master
-#       - next
-#       - beta
-#       - alpha
-# jobs:
-#   release:
-#     name: Release
-#     runs-on: ubuntu-latest
-#     steps:
-#       - name: Checkout
-#         uses: actions/checkout@v3
-#         with:
-#           fetch-depth: 0
-#       - name: Setup Nodejs Env
-#         run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
-#       - name: Install dependencies
-#         run: npm ci
-#       - name: Lint
-#         run: npm run lint
-#       - name: Test
-#         run: npm run test
-#       - name: Release
-#         uses: cycjimmy/semantic-release-action@v2
-#         with:
-#           semantic_version: 16
-#         env:
-#           GITHUB_TOKEN: ${{ secrets.SEMANTIC_RELEASE_GITHUB_TOKEN }}
-#           NPM_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
+name: Release CI
+on:
+  push:
+    branches:
+      - master
+      - next
+      - beta
+      - alpha
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Nodejs Env
+        run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
+      - name: Install dependencies
+        run: npm ci
+      - name: Lint
+        run: npm run lint
+      - name: Test
+        run: npm run test
```

Comment:
The `package.json` file does not specify any `scripts`. Running `npm run lint` or `npm run test` will currently throw an error.

I believe we can remove them since they aren't relevant to this repo. For example, refer to the browserslist-config repo's release.yml workflow file: https://github.com/openedx/browserslist-config/blob/master/.github/workflows/release.yml

Reply:
Good call-out!